android/src/main/java/com/google/android/cameraview/Camera1.java

```diff
@@ -275,7 +275,12 @@ void setPictureSize(Size size) {
         }
         if (mCameraParameters != null && mCamera != null) {
             mCameraParameters.setPictureSize(mPictureSize.getWidth(), mPictureSize.getHeight());
-            mCamera.setParameters(mCameraParameters);
+            try{
+              mCamera.setParameters(mCameraParameters);
+            }
+            catch(RuntimeException e ) {
+              Log.e("Camera1", "setParameters failed", e);
```

Comment:
better keep the same tag for this log

Log.e("RNCamera")

Reply:
you are right, I'm going to change that.

Reply:
Actually it use "CAMERA_1::", should I conserve that?